### PR TITLE
Update Automation Code for CORE OS

### PR DESCRIPTION
**Pull request by iamshital**
Updated Network Tests.

### CODE REVIEW

File: TestLibs/RDFELibs.psm1 [hunk 1]
```diff
@@ -964,8 +966,10 @@ Function SetDistroSpecificVariables($detectedDistro)
 {
 	if ( $detectedDistro -imatch "COREOS" )
 	{
-		$python_cmd = "/usr/share/oem/python/bin/python"
```
Comment (lizzha) [inline TestLibs/RDFELibs.psm1]: this function is not needed now, the if else branches are the same

File: TestLibs/RDFELibs.psm1 [hunk 2]
```diff
@@ -1509,27 +1513,46 @@ Function RemoteCopy($uploadTo, $downloadFrom, $downloadTo, $port, $files, $usern
...
+	
 	if($runAsSudo)
 	{
 		$plainTextPassword = $password.Replace('"','');
-		$linuxCommand = "`"echo $plainTextPassword | sudo -S $command && echo AZURE-LINUX-EXIT-CODE-`$? || echo AZURE-LINUX-EXIT-CODE-`$?`""
-		$logCommand = "`"echo $plainTextPassword | sudo -S $command`""
+		if ( $detectedDistro -eq "COREOS" )
+		{
+			$linuxCommand = "`"export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/share/oem/bin:/usr/share/oem/python/bin:/opt/bin && echo $plainTextPassword | sudo -S env `"PATH=`$PATH`" $command && echo AZURE-LINUX-EXIT-CODE-`$? || echo AZURE-LINUX-EXIT-CODE-`$?`""
+			$logCommand = "`"export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/share/oem/bin:/usr/share/oem/python/bin:/opt/bin && echo $plainTextPassword | sudo -S env `"PATH=`$PATH`" $command && echo AZURE-LINUX-EXIT-CODE-`$? || echo AZURE-LINUX-EXIT-CODE-`$?`""
```
Comment (lizzha) [inline TestLibs/RDFELibs.psm1]: It's not needed the "&& echo AZURE-LINUX-EXIT-CODE-`$? || echo AZURE-LINUX-EXIT-CODE-`$?" for the $logCommand

File: TestLibs/RDFELibs.psm1 [hunk 3]
```diff
@@ -1677,6 +1824,14 @@ Function DoTestCleanUp($result, $testName, $DeployedServices, [switch]$keepUserD
 	{
 		if($DeployedServices)
 		{
+            $currentTestBackgroundJobs = Get-Content $LogDir\CurrentTestBackgroundJobs.txt -ErrorAction SilentlyContinue
+            $currentTestBackgroundJobs = $currentTestBackgroundJobs.Split()
```
Comment (lizzha) [inline TestLibs/RDFELibs.psm1]: exception here "EXCEPTION in DoTestCleanUp : You cannot call a method on a null-valued expression."
should check whether "CurrentTestBackgroundJobs.txt " exists first

**Comment by iamshital** [comment]
Closing this pull request. Will create new pull request with Liz's suggestions..

**Pull request closed without merging**